pom.xml

```diff
@@ -35,6 +35,7 @@
         <automation.framework.version>4.4.3</automation.framework.version>
         <emma.version>2.1.5320</emma.version>
         <synapse.version>2.1.7-wso2v95</synapse.version>
+        <commons.vfs.version.version>2.2-wso2v4</commons.vfs.version.version>
```

Comment:
Can we update to latest 2.2-wso2v6 is used in synapse

Reply:
Fixed in https://github.com/wso2-extensions/esb-connector-file/pull/65